src/controller/java/src/chip/devicecontroller/model/ChipEventPath.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
For the file naming, please consider to remove Chip Prefix since current project is called matter.